addons/payment_alipay/README.md

```diff
@@ -0,0 +1,25 @@
+# Alipay
+
+## Technical details
+
+API: [Global API](https://global.alipay.com/docs/ac/global/create_forex_trade) that is part of the
+[cross-border website payment solution](https://global.alipay.com/docs/ac/web/integration)
+
+This module integrates Alipay using `payment`'s form-based implementation of the payment with
```

Comment:
This is not equivalent because I was referring to "form data", not the payment form.

I'm okay to rephrase it, though. What about:
> This module integrates Alipay using the generic payment with redirection flow based on form
submission provided by the `payment` module.

Reply:
I think it is indeed more understandable.

Reply:
Maybe some articles are missing.
> This module integrates Alipay using the generic payment with **a** redirection flow based on **the** form
submission provided by the payment module.

Reply:
No, because neither the redirection nor the form are specific in this context.